src/sections/create-dataset/AuthorFormGroup.tsx

```diff
@@ -0,0 +1,71 @@
+import { Form } from '@iqss/dataverse-design-system'
+import { useTranslation } from 'react-i18next'
+import { SubmissionStatus } from './useCreateDatasetForm'
+import { DatasetMetadataSubField } from '../../dataset/domain/models/Dataset'
+import { Col, Row } from '@iqss/dataverse-design-system'
+import { DynamicFieldsButtons } from './dynamic-fields-buttons/DynamicFieldsButtons'
+
+import { ChangeEvent } from 'react'
+import _ from 'lodash'
+import { useMultipleFields } from './useMultipleFields'
+interface AuthorFormGroupProps {
+  submissionStatus: SubmissionStatus
+  initialAuthorFields: DatasetMetadataSubField[]
+}
+
+export function AuthorFormGroup({ submissionStatus, initialAuthorFields }: AuthorFormGroupProps) {
```

Comment:
I believe we need to pass updateFormData as a prop and invoke it within handleFieldChange, since this function is responsible for updating the form data

Reply:
thanks for your comments, @MellyGray I made the changes you suggested